addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComTemperatureHumidityMessage.java

```diff
@@ -90,7 +109,7 @@ public byte toByte() {
 
     private final static List<RFXComValueSelector> supportedOutputValueSelectors = Arrays.asList();
 
-    public SubType subType = SubType.UNDEF;
+    public SubType subType = SubType.TH1;
```

Comment:
I agree with removing the undef, especially if it's not in spec, but they all also have UNKNOWN(255). I thought initializing with that might be more accurate.